packages/terra-search-field/tests/jest/__snapshots__/SearchField.test.jsx.snap

```diff
@@ -53,16 +57,18 @@ exports[`Snapshots renders a search field with a placeholder 1`] = `
 exports[`Snapshots renders a search field with a value 1`] = `
 <div
   className="terra-SearchField">
-  <input
+  <Input
     className="terra-SearchField-input"
+    name={null}
     onChange={[Function]}
     placeholder=""
+    required={false}
```

Comment:
The default value for required in [Input](https://github.com/cerner/terra-core/blob/master/packages/terra-form/src/Input.jsx#L41) is false so I just let use be the default.